jobs/docker/templates/bin/docker_ctl.erb

```diff
@@ -43,6 +43,7 @@ case $1 in
         fi
         if ! grep -Fxq "\$ModLoad imtcp" /etc/rsyslog.d/10-fabrik_modules.conf ; then echo "\$ModLoad imtcp" >> /etc/rsyslog.d/10-fabrik_modules.conf ; fi
         if ! grep -Fxq "\$ModLoad imfile" /etc/rsyslog.d/10-fabrik_modules.conf ; then echo "\$ModLoad imfile" >> /etc/rsyslog.d/10-fabrik_modules.conf ; fi
+        if ! grep -Fxq "\$InputTCPServerRun 1514" /etc/rsyslog.d/10-fabrik_modules.conf ; then echo "\$InputTCPServerRun 1514" >> /etc/rsyslog.d/10-fabrik_modules.conf ; fi
```

Comment:
If the file already exists in the VM, the initial check does not allow to do changes in it. Could you please handle that as well?

Reply:
Also, as discussed over email, would you like to keep it in one job and remove from everywhere else?

Reply:
Hi,
I combined the duplicated Configuration-Files to a single one. I also removed the File-Check for the Entry-Check, so the presence of the File should no longer prevent the required entry’s from being added to `10-fabrik_modules.conf`
Best regards